src/download.ts

```diff
@@ -1,18 +1,75 @@
 import * as config from 'config';
 import { isDireectoryExists, mkdir, writeFile, isFileExists } from './helpers';
-import { ROOT_PATH, HTTP_REQUEST_TIMEOUT, PageTitleAndLink, SAVE_LESSON_AS } from './globals';
+import { ROOT_PATH, HTTP_REQUEST_TIMEOUT, PageTitleAndLink, SAVE_LESSON_AS, AvailableCourses } from './globals';
 import { getPage, getSpecialBrowser } from './browser';
 import { Browser, Page } from 'puppeteer';
 
 let SAVE_DESTINATION = '';
 const SAVE_AS: string = config.get('saveAs');
 const MULTI_LANGUAGE: boolean = config.get('multiLanguage');
+const COURSE_URL_SLUG_LIST = [];
```

Comment:
Can make it constant but it can't be moved inside fetchAllCoursesAvailableToDownload() because it is a recursive function and it will be re-initialized in that case, which we don't want.

@shihabmridha - `Your Comment Was - Make it const and local variable of fetchAllCoursesAvailableToDownload().`